ci/docker/linux-apt-python-3.dockerfile

```diff
@@ -36,7 +36,8 @@ COPY python/requirements.txt \
 RUN pip install \
     -r arrow/python/requirements.txt \
     -r arrow/python/requirements-test.txt \
-    cython setuptools
+    "pickle5; python_version == '3.6' or python_version == '3.7'" \
```

Comment:
Can we add this to `requirements-test.txt`?

Reply:
Done.

Reply:
Thanks!